src/sidebar/media-embedder.js

```diff
@@ -332,6 +370,14 @@ export function replaceLinksWithEmbeds(element) {
 
   let i;
   for (i = 0; i < links.length; i++) {
-    replaceLinkWithEmbed(links[i]);
+    const embed = replaceLinkWithEmbed(links[i]);
+    if (embed) {
+      if (className) {
+        embed.className = className;
+      } else {
+        // Default width.
+        embed.style.width = '350px';
```

Comment:
Does this mean that if one doesn't provide a `className` that the embed won't be so great on narrow viewports?

Reply:
Indeed. We currently only use this function in one place where we do set `className` though.